packages/tempest/src/TextTest.test.tsx

```diff
@@ -0,0 +1,20 @@
+import {cleanup} from 'react-testing-library'
+import {Text} from './Text'
+import * as React from 'react'
+import * as enzyme from 'enzyme'
+import * as Adapter from 'enzyme-adapter-react-16';
```

Comment:
It looks like prettier isn't running here- was there an issue with it?

Reply:
To be honest I'm not sure - I was very focused on trying to get the functionality of the test first. I can try to debug why its not working when I get a chance